module/proto/gravity/v1/msgs.proto

```diff
@@ -19,9 +19,6 @@ service Msg {
       returns (MsgCancelSendToEthereumResponse) {
     // option (google.api.http).post = "/gravity/v1/send_to_ethereum/cancel";
   }
-  rpc RequestBatchTx(MsgRequestBatchTx) returns (MsgRequestBatchTxResponse) {
```

Comment:
Can we double check that the web UI isn't calling this?

Reply:
Confirmed with Elliot, who confirmed with Strangelove.